payjoin/src/core/uri/v2.rs

```diff
@@ -434,6 +434,26 @@ mod tests {
             ohttp(&invalid_ohttp_url),
             Err(ParseOhttpKeysParamError::InvalidFragment(_))
         ));
+
+        let invalid_ohttp_url =
```

Comment:
Whats the difference between these two urls? Its not immediately clear.
Ones too long other too short? Nit: a comment or asserting the inner error message would self-document this test.